cloudinstall/controllers/installbase.py

```diff
@@ -159,5 +162,7 @@ def start(self):
             self.ui.select_install_type(
                 self.config.install_types(), self._set_install_type)
 
-        self.update()
+        self.alarm = self.loop.set_alarm_in(1, self.update)
+        self.observe('stop alarm', partial(self.loop.remove_alarm,
+                                           self.alarm))
```

Comment:
Quick question, I don't have this all fully in my head right now:
Why change this from self.update? that was doing more than just the alarm setup…